### DISCUSSION TITLE
Modify Start "See more results"

### DISCUSSION THREAD

**Discussion by joram467456** [Open-Shell/Open-Shell-Menu#1702] [Q&A]
I'd like to know if and where I can go to change the See more results
- I want to change the text and icon.
- Open the application Everything instead of Indexed Locations.

I'm new to coding so I expect things to a bit advanced for me. Will this cause things to break?

Click here
![image](https://github.com/Open-Shell/Open-Shell-Menu/assets/145044410/5baf1367-ec02-4fd0-9c20-b00d4a23e728)

to open
![image](https://github.com/Open-Shell/Open-Shell-Menu/assets/145044410/72061b6e-5cd8-48c7-bf3f-03c81d4cef92)

instead of this.
![image](https://github.com/Open-Shell/Open-Shell-Menu/assets/145044410/179befcc-d6b1-4652-9ad9-8a85fd7eacbe)

**Comment by ge0rdi** [comment]
Searching for [See more results](https://github.com/search?q=repo%3AOpen-Shell%2FOpen-Shell-Menu%20%22See%20more%20results%22&type=code) reveals two locations in code that refer to it:

- https://github.com/Open-Shell/Open-Shell-Menu/blob/8359cfcc81f1247a8299dcfd0c9bbed339796aae/Src/StartMenu/StartMenuDLL/MenuCommands.cpp#L2376
- https://github.com/Open-Shell/Open-Shell-Menu/blob/8359cfcc81f1247a8299dcfd0c9bbed339796aae/Src/StartMenu/StartMenuDLL/MenuContainer.cpp#L2975

First location inserts `CMD_EXPLORE` item to search list.
Second location adds actual icon to the item.

Searching for [CMD_EXPLORE](https://github.com/search?q=repo%3AOpen-Shell%2FOpen-Shell-Menu+%22CMD_EXPLORE%22&type=code) yields:
https://github.com/Open-Shell/Open-Shell-Menu/blob/8359cfcc81f1247a8299dcfd0c9bbed339796aae/Src/StartMenu/StartMenuDLL/MenuCommands.cpp#L2857
Few lines below it can be seen that `g_SearchManager.LaunchExternalSearch` is executed.

So this is the place that starts actual search:
https://github.com/Open-Shell/Open-Shell-Menu/blob/8359cfcc81f1247a8299dcfd0c9bbed339796aae/Src/StartMenu/StartMenuDLL/SearchManager.cpp#L1499
